src/engine/controls/cuecontrol.cpp

```diff
@@ -498,6 +507,10 @@ void CueControl::loadCuesFromTrack() {
             }
             // Add the hotcue to the list of active hotcues
             active_hotcues.insert(hotcue);
+            break;
+        }
+        default:
+            continue;
```

Comment:
can we DEBUG_ASSERT(!"unknown cue type") or such?

Reply:
I don't think so. The loop goes over all cue points regardless of type. I think the other types (Beginning, AudibleSound, etc.) are supposed to be silently ignored.